tests/unit/web/test_models.py

```diff
@@ -0,0 +1,57 @@
+from django.test import TestCase
+from django.core.files.uploadedfile import SimpleUploadedFile
+
+from web.models import Contact, Team
+
+
+class BaseTestCase(TestCase):
```

Comment:
If BaseTestCase is doing nothing, we can simply skip this and derive all the other class from TestCase.